resources/assets/js/components/GroupEventSummary.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-tr v-if="event && stats" :class="{
```

Comment:
Not sure what these changes are - are the unrelated to the landing page - or is this component now also used on the landing page?

Reply:
Fixing some errors - we'd get JS errors if stats is null.